lib/pact/api/exec_command_request.ex

```diff
@@ -0,0 +1,315 @@
+defmodule Kadena.Pact.API.ExecCommandRequest do
+  @moduledoc """
+  Specifies functions to build PACT execution command requests.
+  """
+
+  alias Kadena.Chainweb.Pact.JSONPayload
+  alias Kadena.Cryptography.Sign
+  alias Kadena.Pact.API.CommandRequest
+
+  alias Kadena.Types.{
+    CapsList,
+    Command,
+    CommandPayload,
+    EnvData,
+    ExecPayload,
+    KeyPair,
+    MetaData,
+    NetworkID,
+    OptionalCapsList,
+    PactPayload,
+    SignaturesList,
+    SignCommand,
+    Signer,
+    SignersList
+  }
+
+  @behaviour CommandRequest
+
+  @type arg :: {arg_type(), arg_value()}
+  @type arg_type :: atom()
+  @type arg_value :: key() | clist()
+  @type arg_validation :: {:ok, arg_value()} | {:error, Keyword.t()}
+  @type clist :: OptionalCapsList.t()
+  @type cmd :: String.t()
+  @type code :: String.t()
+  @type command :: Command.t()
+  @type data :: EnvData.t() | nil
+  @type hash :: String.t()
+  @type key :: String.t()
```

Comment:
@einerzg what would be the appropriate standards for these types ?
Because the criteria for a different project were arranged alphabetically.

Reply:
@Odraxs, the order proposed by @einerzg , it is ordered according to the use of those types in other types, so that it is easy to read them since the types used are already defined previously.